src/app/pages/MediaArticlePage/PagePromoSections/LatestMediaSection/index.styles.tsx

```diff
@@ -0,0 +1,49 @@
+import pixelsToRem from '#app/utilities/pixelsToRem';
+import { Theme, css } from '@emotion/react';
+
+const styles = {
+  latestMediaSection: ({ mq, isDarkUi, palette }: Theme) =>
+    css({
+      backgroundColor: isDarkUi ? palette.GREY_10 : palette.GREY_2,
+      margin: '0 0.5rem',
```

Comment:
Just checked with @emilymynall, this margin should only apply on the mobile breakpoint when it goes back to a single column of promo items.

Reply:
I have resolved by adding a [mq.GROUP_4_MIN_WIDTH]: media query to remove the margin. I decided to do this as it follows 'mobile' first approach. 

An alternative would be to apply this margin only to 0/1/2 breakpoints, and then make use of the tablet auto margin.

Let me know if you think I should update it.

Reply:
What you have now looks good to me!